Cargo.toml

```diff
@@ -30,7 +30,7 @@ name = "prettytable"
 [dependencies]
 unicode-width = "^0.1"
 term = "^0.4"
-lazy_static = "^0.2"
+lazy_static = "1"

```

Comment:
Shouldn't it be `^1` ?

Reply:
From http://doc.crates.io/specifying-dependencies.html#specifying-dependencies-from-cratesio:

>Cargo is configured to look for dependencies on crates.io by default. Only the name and a version string are required in this case. In the cargo guide, we specified a dependency on the time crate:
>
>```toml
>[dependencies]
>time = "0.1.12"
>```
>
>The string "0.1.12" is a semver version requirement. Since this string does not have any operators in it, it is interpreted the same way as if we had specified "^0.1.12", which is called a caret requirement.

So, this is essentially the same.

Reply:
Didn't know that. So it looks good to me. Thanks for updating both the crate and my knowledge